### PR TITLE
Align page structure of all @@iterator pages

### PR DESCRIPTION
**Pull request by Josh-Cena**
### Description

This should make maintenance much easier; also filled in some missing information.

### Motivation

There's some confusing wording about how method aliasing works: `@@iterator` doesn't _return_ the `values()` method; it _is_ the `values()` method.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md [hunk 1]
```diff
@@ -79,36 +72,30 @@ console.log(arrIter.next().value); // d
...
 
 ```js
 function logIterable(it) {
   if (!(Symbol.iterator in it)) {
-    console.log(it, ' is not an iterable object.');
+    console.log(it, " is not an iterable object.");
     return;
   }
 
-  const iterator = it[Symbol.iterator]();
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:98]: Is this line removed on purpose?

**Review by teoli2003 — DISMISSED** [review]

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:98]: Yes, as explained by the paragraph above, you rarely explicitly call `Symbol.iterator` itself—you call it via various iterating syntaxes.

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md [hunk 2]
```diff
@@ -79,36 +72,30 @@ console.log(arrIter.next().value); // d
...
   for (const letter of iterator) {
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:86]:
```suggestion
  for (const letter of it) {
```

File: files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/array/@@iterator/index.md:98]: Oh, oops! I see what you are talking about. I've fixed the next line.

**Review by wbamberg — APPROVED** [review]
👍 AFAICT you addressed Jean-Yves' comment, so I'm happy to merge this if you can fix the conflicts. Thank you @Josh-Cena !

**Pull request merged**